providers/src/airflow/providers/cncf/kubernetes/utils/pod_manager.py

```diff
@@ -832,6 +883,20 @@ def _exec_pod_command(self, resp, command: str) -> str | None:
                 return res
         return None
 
+    def _await_init_container_start(self, pod: V1Pod, container_name: str, sleep_timeout_seconds: float = 1):
```

Comment:
if nothing is using this param, and it's a private function anyway, just remove the param

Reply:
Would you like to have `sleep_timeout_seconds` as const? Which param are unused?

Reply:
i was just saying it seems you don't use the param `sleep_timeout_seconds` so it probably doesn't need to be there. not saying you should make it a constant but it could just be a hardcoded number. just the principle of not adding interface that you aren't actually using / doing the minimal thing.  you've made it a private function (which is great) so this means if we want to change it later we can.

Reply:
Fixed